CPl/README.md

```diff
@@ -0,0 +1,7 @@
+# Information for CAMPLA Exams
+This summary contains links to cppreference.com. This site is available offline in the CAMPLA exam environment. To ensure that the links also work in the exam, there is an “EXAM_MODE” flag in the `00_CPR_Settings.typ` file. Setting it to true converts the links to the offline versions.
+
+If you don't want to customize the summary yourself, this repository contains two PDF's, one with online links and one with offline links.
+
+It is however worth checking whether the offline version is still in the same place when testing the exam environment ;)
```

Comment:
whether the offline version is still in the same path during the CAMPLA testing session (scheduled during a CPl exercise session in the last weeks of the semester)